src/app/lib/utilities/passport/index.js

```diff
@@ -20,15 +19,15 @@ export const isValidPassportHome = (
   service,
   passportHomesOverride = [],
 ) => {
-  const passportHomesOverrideArray = passportHomesOverride || [];
-  const passportHomeLower = (passportHome || '').toLowerCase();
+  const isMissingRequiredArgs = !passportHome;
+  if (isMissingRequiredArgs) return true;
 
-  if (!passportHome) return true;
+  const matchesPassportHome = (home = '') =>
```

Comment:
the default arg value `(home || '')` can be removed if you are doing `(home || '')` on the line below

Reply:
Fixed.